avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -0,0 +1,50 @@
+### Config File - This file contains the main settings for the range time diagram in distanceTimeAnalysis
+## Copy to  local_distanceTimeAnalysisCfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# result type used for creating range time diagram e.g. FD, FV, ....
+rangeTimeResType = FD
+# time steps used for creating diagram
+tSteps = 0:1
+# threshold - below this result data is ignored - used to create mask
+# average value created of where there is avalanche flow (threshold > 0.0)
+# this is default set to 1.e-5 to account for any spurious values
+# also a different threshold can be set e.g. if only regions with > 0.5m FD are of interest
+thresholdResult = 1.e-5
+
+#+++++range time diagram with radar field of view +++++
+# width of range gates in meters
+rgWidth = 5.
+# aperture angle in degrees
+aperture = 15
+
+# radar location field of view points min (x0, y0) and point giving direction (x1, y1)
+# format: x0|x1|y0|y1 needs to be in same projection as DEM
+radarLocation =
+
+#++++++++thalweg time diagram along avalanche path ++++++++++++
+# domain width orthogonal to avalanche path
+domainWidth = 200
+
+# The cell size for the new (s,l) raster is automatically computed from the reference result file (leave the following field empty)
+# It is possible to force the cell size to take another value, then specify this new value below, otherwise leave empty (default).
+cellSizeSL =
+
+# angle for the start of the run-out zone
+startOfRunoutAreaAngle = 10
+
+# chose interpolation method between 'nearest' and 'bilinear'
+interpMethod = bilinear
+
+# threshold distance [m]. When looking for the beta point make sure at least
+# dsMin meters after the beta point also have an angle bellow 10°
+dsMin = 30
```

Comment:
These are all related to AIMEC, right?. Why not use the default configuration from that module instead of duplicating it here?

Reply:
decided to keep this